zerver/webhooks/runscope/doc.md

```diff
@@ -0,0 +1,27 @@
+Receive alerts on whenever your Runscope tests run!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+{!append-topic.md!}
+
+As well as changing the topic used by the bot, you can
+also tell the bot to only send a message when a particular
+build fails or passes by appending `&only_on=fail` or
+`&only_on=pass` respectively.
+
+For example, a webhook URL that uses the API KEY `abcdefgh`,
+sends messages to the stream `Runscope Tests` and topic `API Tests 1`,
+only when a build fails would look like:
+
+`{{api_url}}?api_key=abcdefgh&stream=Runscope%20Tests&topic=API%20Tests%201&only_on=fail`
+
+Now that you've got your webhook URL, go to your test settings, and add
+your URL to the webhook section:
```

Comment:
If `test settings` and `webhook section` are actual physical locations on the Runscope website, make sure you properly bold and capitalise them like `**Webhook** section`. We bold the names of sections in third-party websites. The images you included don't seem to be getting rendered due to some issue with GitHub so I can't see what you are referring to, so this comment might not even make sense! :D